src/annotator/test/guest-test.js

```diff
@@ -93,7 +84,6 @@ describe('Guest', () => {
     }
 
     CrossFrame = sandbox.stub().returns(fakeCrossFrame);
-    guestConfig.pluginClasses.CrossFrame = CrossFrame;
```

Comment:
This was a method of mocking `CrossFrame` in these tests. Replaced by a more conventional method of mocking below.